### PR TITLE
Remove all code related to Direct Deposit v1

### PR DESCRIPTION
**Pull request by erikphansen**
## Description
This PR removes all code related to the Direct Deposit component that existed _before_ we added Direct Deposit for Education Benefits.

- This includes removing any usage of the dd4edu feature flag.
- Also included deleting the DirectDepositContent React widget (which was added in [this commit](https://github.com/department-of-veterans-affairs/vets-website/commit/8864daff875886046da9e86d8110ddd6b71b6409)) that used to be embedded on https://www.va.gov/change-direct-deposit/

## Testing done
Local + existing tests, a small number of which required minor updates.

## Screenshots
N/A

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/personalization/profile/components/alerts/DirectDepositConnectionError.jsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import AlertBox from '@department-of-veterans-affairs/component-library/AlertBox';
 
-import { benefitTypes } from '../direct-deposit/DirectDepositV2';
```
Comment (erikphansen) [inline src/applications/personalization/profile/components/alerts/DirectDepositConnectionError.jsx:4]: `DirectDepositV2` was renamed `DirectDeposit`

File: src/applications/personalization/profile/components/direct-deposit/BankInfoCNP.jsx [hunk 2]
```diff
@@ -3,65 +3,66 @@ import PropTypes from 'prop-types';
 import { Prompt } from 'react-router-dom';
```
Comment (erikphansen) [inline src/applications/personalization/profile/components/direct-deposit/BankInfoCNP.jsx:3]: Sorry about all the diff noise in this file. What _was_ BankInfoCNPv2 is now BankInfoCNP, so you can basically ignore any "changes" that are shown here.

File: src/applications/static-pages/widgetTypes.js [hunk 3]
```diff
@@ -6,7 +6,6 @@ export default {
   CORONAVIRUS_CHATBOT: 'va-coronavirus-chatbot',
   COVID_VACCINE_UPDATES_CTA: 'va-covid-vaccine-updates-cta',
   CTA: 'cta',
-  DIRECT_DEPOSIT: 'direct-deposit',
```
Comment (erikphansen) [inline src/applications/static-pages/widgetTypes.js:9]: I confirmed that this react widget is no longer used on https://www.va.gov/change-direct-deposit/

**Review by micahchiang — APPROVED** [review]

**Review by kelsonic — APPROVED** [review]

**Comment by erikphansen** [comment]
@department-of-veterans-affairs/frontend-review-group Rats. Sorry, I didn't realize that I was going to need a review from y'all on this PR so I didn't leave it as a draft PR while I collected reviews from other teams. Anyhow, it's ready for you now.

**Review by oseasmoran73 — APPROVED** [review]

**Pull request merged**